### DISCUSSION TITLE
Vitest Integration

### DISCUSSION THREAD

**Discussion by Davide-Gheri** [automock/automock#158] [Ideas]
Hi contributors, first of all, great work on this project! It is really useful!

Now the feature request: Vitest is growing in popularity and even [Nestjs](https://docs.nestjs.com/recipes/swc#vitest) supports it right now, it would be great to use @automock with it.

Since Vitest's API is almost identical to Jest's, it should be trivial to create a new package for it.

**Comment by omermorad** [comment]
Hello @Davide-Gheri,

Thank you! And I appreciate you mentioning this 🚀 

Indeed, Vitest is becoming more and more popular. We intend to add support for it in the near future (under `@automock/vitest`), however this is likely to happen after Automock 2.0 is released :)

I hope to post an update here soon!
Thanks.

**Comment by tednaaa** [comment]
Hey guys, is there any update with vitest integration? I'd like to use it

**Comment by omermorad** [comment]
Hi everyone,

We're excited to announce that Automock has rebranded into Suites, and I'm happy to share that Suites now supports Vitest! 🎉
Looking forward to your feedback 🚀

Here is a link for the installation guide: https://suites.dev/docs/overview/installation

CC: @Davide-Gheri

**Reply by tednaaa** [reply]
Great job, thank you
